core/driver/sources/src/main/scala/org/virtuslab/ideprobe/wait/ConstantWaiting.scala

```diff
@@ -0,0 +1,27 @@
+package org.virtuslab.ideprobe
+package wait
+
+import scala.concurrent.duration.FiniteDuration
+
+class ConstantWaiting private (
+    duration: FiniteDuration,
+    waitCondition: ProbeDriver => WaitDecision
+) extends WaitLogic {
+
+  def this(duration: FiniteDuration) = {
+    this(duration, _ => WaitDecision.Done)
+  }
+
+  override def await(driver: ProbeDriver): Unit = {
+    waitCondition(driver)
```

Comment:
We name this param `waitCondition` but we don't check its result

Reply:
This is constant waiting. I execute condition for consistency and potential side effects. It waits for constant time, no matter what.